infrastructure/resource_group_level/modules/gef-ui.bicep

```diff
@@ -0,0 +1,219 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param logAnalyticsWorkspaceId string
+param externalApiHostname string
+param tfmApiHostname string
+param portalApiHostname string
+param redisName string
+
+var dockerImageName = '${containerRegistryName}.azurecr.io/gef-ui:${environment}'
+var dockerRegistryServerUsername = 'tfs${environment}'
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureSettings object = {
+
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+@secure()
+param additionalSecureSettings object = {
+  DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'
+  DTFS_CENTRAL_API_KEY: 'test-value'
+  EXTERNAL_API_KEY: 'test-value'
+  PORTAL_API_KEY: 'test-value'
+  TFM_API_KEY: 'test-value'
+}
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureConnectionStrings object = {
+  COMPANIES_HOUSE_API_KEY : 'test-value'
```

Comment:
interesting. we need to check if this is actually used because GEF-UI shouldn't call the Companies House API directly

Reply:
I've marked for removal in FN-820